### DISCUSSION TITLE
Shall we still stick with Gradle 4.10.3 to build Cordova project?

### DISCUSSION THREAD

**Discussion by xiaoxiao99** [quasarframework/quasar#13633] [CLI - Capacitor / Cordova modes]
Quasar V2.7.1 [help page](https://quasar.dev/quasar-cli-webpack/developing-cordova-apps/preparation#android-setup) said
**There is a very specific version that cordova requires. You can download it [here](https://downloads.gradle-dn.com/distributions/gradle-4.10.3-all.zip)**

Where the link points to Gradle 4.10.3.

However, it seems the latest cordova android project has [updated](https://github.com/apache/cordova-android/pull/125) to support Gradle7.1.1

So the questions is, shall we stick with Gradle 4.10.3, or update the help page instead?

**Comment by metalsadman** [comment]
haven't built a cordova app in a while, you should try it see if new gradle version is working.
